packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -273,12 +314,28 @@ function CategoryHeader({ category }: { category: CategoryWithDapps }) {
   )
 }
 
+function FeaturedDapp({ dapp, onPressDapp }: DappProps) {
+  const onPress = () => onPressDapp(dapp)
+
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.Soft}>
+      <TouchableOpacity style={styles.pressableCard} onPress={onPress} testID="featuredDapp">
```

Comment:
nit: I think we've been mainly using Capitalized strings for testIDs, though there are some inconsitencies. 👀 
```suggestion
      <TouchableOpacity style={styles.pressableCard} onPress={onPress} testID="FeaturedDapp">
```